include/seng499/measurement_pipeline_block.hpp

```diff
@@ -34,6 +76,34 @@ namespace seng499 {
 			 *	pointer or reference to base.
 			 */
 			virtual ~measurement_pipeline_block () noexcept;
+			
+			
+			/**
+			 *	Obtains vertex and normal maps for a certain
+			 *	depth frame.
+			 *
+			 *	\param [in] frame
+			 *		A \ref pipeline_value representing the depth
+			 *		frame.
+			 *	\param [in] width
+			 *		The width of \em frame.
+			 *	\param [in] height
+			 *		The height of \em frame.
+			 *	\param [in] k
+			 *		The \f$K\f$ matrix for the \ref depth_device
+			 *		which generated \em frame.
```

Comment:
The width, heigh and K should be associated to our depth_device. Should we just pass that in instead?